### PR TITLE
text and list component created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
Web capture_1-2-2022_214347_127 0 0 1

https://user-images.githubusercontent.com/73058928/152116189-6e0376a0-37e9-4f14-bdea-fe416c3d65c5.mp4

### CODE REVIEW

**Comment by tanishk15sharma** [comment]
looking good , just a suggestion you can use color variables to give colors .

**Comment by ivinayakg** [comment]
> looking good , just a suggestion you can use color variables to give colors .

**Review by sakshi006 — APPROVED** [review]
Good to go.

File: COMPONENTS/list/index.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+.list_comp--row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
+.list_heading {
+  padding: 4.5px 3rem 4.5px 11px;
```
Comment (chanchal16) [inline COMPONENTS/list/index.css:23]:
```
Suggestion: Don't use mixed units, stick to one only. 
```

Comment (ivinayakg) [reply COMPONENTS/list/index.css:23]: i will keep that in mind thank you

**Pull request merged**